### PR TITLE
MCKIN-24382 Scorm VB

### PR DESCRIPTION
**Pull request by musmanmalik**
Scorm VB

### CODE REVIEW

File: requirements/edx/custom.txt [hunk 1]
```diff
@@ -16,8 +16,8 @@ git+https://github.com/edx-solutions/xblock-group-project.git@0.1.3#egg=xblock-g
 -e git+https://github.com/open-craft/problem-builder.git@v3.5.7#egg=xblock-problem-builder==3.5.7
 -e git+https://github.com/OfficeDev/xblock-officemix.git@86238f5968a08db005717dbddc346808f1ed3716#egg=xblock-officemix
 -e git+https://github.com/open-craft/xblock-chat.git@v0.2.8#egg=xblock-chat==0.2.8
--e git+https://github.com/open-craft/xblock-eoc-journal.git@v0.9.1#egg=xblock-eoc-journal==0.9.1
--e git+https://github.com/mckinseyacademy/xblock-scorm.git@v2.2#egg=xblock-scorm==2.2
+-e git+https://github.com/open-craft/xblock-eoc-journal.git@v0.9.2#egg=xblock-eoc-journal==0.9.2
+-e git+https://github.com/mckinseyacademy/xblock-scorm.git@v3.1#egg=xblock-scorm==3.1
```
Comment (ihtram) [inline requirements/edx/custom.txt:20]: @musmanmalik is there any reason to jump from 2.2 to 3.1

Comment (musmanmalik) [reply requirements/edx/custom.txt:20]: @ihtram there was a jump from 2.2 to 3 while adding 3 python 3.5 support. So 3.0 already released but not updated here.

Comment (musmanmalik) [reply requirements/edx/custom.txt:20]: https://github.com/mckinseyacademy/xblock-scorm/releases

**Review by ihtram — APPROVED** [review]
👍

**Pull request merged**